OpenSim/Simulation/OpenSense/OpenSenseUtilities.h

```diff
@@ -56,9 +63,20 @@ namespace OpenSim {
                 const SimTK::Array_<int>& startEnd = { 0, 1 },
```

Comment:
SimTK::Array_<int> translates to SimTKArrayInt (in Matlab) and has no set or upd methods that are scripting accessible, so I would like to suggest;
(1) the input data type change to something useable in scripting (ArrayDouble?) 
(2) the default end time is the full size of the input table